packaging/README.md

```diff
@@ -0,0 +1,33 @@
+# Distribute and Install nmstate
+The packaging and distribution of nmstate is currently implemented through
+PyPI and Docker images.
+
+First prepare and upload to PyPI and then build and upload the docker image.
+ 
+
+## Package and Upload to PyPI
+The following procedures can be reviewed in detail [here](https://packaging.python.org/tutorials/packaging-projects/#uploading-the-distribution-archives).
+```
+cd <project-path>
+python setup.py sdist bdist_wheel
+twine upload dist/*
+```
+
+## Build a new container image and push to the docker hub
+
+- CentOS 7:
+```
+cd <project-path/packaging>
+sudo docker build --rm -t local/centos7-nmstate .
+docker tag local/centos7-nmstate nmstate/centos7-nmstate:<ver>
```

Comment:
What is the value of `<ver>` supposed to be? Also IMHO it would be better to write a shell or python script for creating a release instead of documenting the steps in a README.

Reply:
`<ver>` is the version number.
A script will be nice, but I do not think it is a must at the moment. I am not that focused on the release process at the moment, it will happen eventually as it is the right thing to do.

Reply:
But how do we choose the version number? If I had to follow these instructions, I would not know which number would be correct. The shell script can be just all commands in a single file with `set -e`, so it would not be much extra work but it makes it easier for us to test this change.

Reply:
How about we just use lastest for the docker version for now?

Reply:
There seems to be a 0.1 tag already, I suppose this should be removed: https://hub.docker.com/r/nmstate/centos7-nmstate/tags/

Reply:
`latest` and `0.1` are the same. This is a way to track what `latest` means.
With NMState we use 0.1bXX where XX is the build num (it could have been a commit hash like we do with other projects).

Reply:
0.1 means it is the first one that works :)
0.1b1 is the next one.

We need to user of this image to know which `latest` he is using. Latest is just a pointer to the last version.

I expect to release it every time we update the one on PyPI.
Again, I think it belongs to a different PR as it discusses a release flow/procedure. And we do not have a release process yet.